packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/WorkflowSelector.js

```diff
@@ -37,18 +37,26 @@ function WorkflowSelector (props) {
 
       {(workflows.loading === asyncStates.success) && (
         <Box margin={{ top: 'small' }}>
-          {workflows.data.map(workflow =>
+          {(workflows.data.length > 0) && workflows.data.map(workflow =>
             <WorkflowSelectButton key={workflow.id} workflow={workflow} />
           )}
+          {(workflows.data.length === 0) && (
+            <WorkflowSelectButton
```

Comment:
Can this be a styled Box? A button that's never intended to be clicked seems like the wrong element to use here.